### PR TITLE
Add a highcharts module to include html icon in ..

### PR DESCRIPTION
**Pull request by lingyun1010**
export button text and enlarge the font size

### CODE REVIEW

File: html/index.html [hunk 1]
```diff
@@ -14,6 +15,10 @@
             box-shadow: none;
             margin: 0;
         }
+
```
Comment (monicajianu) [inline html/index.html:18]: Same comments as in the experiment page PR, I don't think this CSS will be applied unless you use styled mode.

Comment (lingyun1010) [reply html/index.html:18]: The css is applied here, which enables the download button to be triggered by clicking *on* text and icon. Apart from this comment, I have added a source link for the highcharts module and set the font for both button text and item menu as `15px`. As heatmap is using extra download options, I keep the item lists in the download menu here.

**Review by alfonsomunozpomer — CHANGES_REQUESTED** [review]
Same comments as in https://github.com/ebi-gene-expression-group/scxa-experiment-page-tsne-plot-view/pull/23 apply here.

File: src/MarkerGeneHeatmap.js [hunk 2]
```diff
@@ -242,7 +242,7 @@ const MarkerGeneHeatmap = (props) => {
     exporting: {
       buttons: {
         contextButton: {
-          text: '<i class="icon icon-functional" data-icon="="></i> Download &nbsp;',
+          text: '<i class="icon icon-functional" data-icon="="></i>Download&nbsp;',
```
Comment (alfonsomunozpomer) [inline src/MarkerGeneHeatmap.js]: The space should be **between** the `<i>` element and the *Download* label: `<i ...></i>&nbsp;Download`

**Review by alfonsomunozpomer — APPROVED** [review]
Good! Remember to change the non-breaking space position and merge away.

**Pull request merged**